### ISSUE TITLE
use constant INIT failure

### ISSUE THREAD

**Issue by toddr** [rurban/perl-compiler#238]
```
perlcc -c -e'sub is { $_[0] eq $_[1] and print "ok\n"} use constant INIT => 5; is(INIT, 5)'
```
This works on cm522.

**Comment by rurban** [comment]
I tested it back, and it fails for all my compiler versions with 5.14.4. How did you manage to fix that? Do you have an idea?

**Comment by toddr** [comment]
see pull 240

**Comment by toddr** [comment]
fixed with 0149ba550654c254b8159843f9eb5b84e9d483a4
